tests/ui/question_mark.rs

```diff
@@ -1,5 +1,6 @@
 #![feature(try_blocks)]
 #![allow(clippy::unnecessary_wraps)]
+#![allow(dropping_copy_types)]
```

Comment:
No tests requires this AFAICS.

Can you add a test similar to the one you add but with a `Result<i32, ()>` (or any type that implements `Copy`?)